docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers Continuous Integration(CI) support for projects hosted on
+GitHub. This service is called CIFuzz and can be used to run OSS-Fuzz fuzz targets
+on your project when a pull request is submitted. It has the advantage of
+detecting bugs before they are committed into your project. This insures that
```

Comment:
s/insures/ensures/.

Also these two sentences can probably be combined into one along the lines of:

"CIFuzz complements OSS-Fuzz by finding more bugs earlier before they get committed at all".

Reply:
Fixed.